### DISCUSSION TITLE
Sunrise-Sunset Brightness Adjustments

### DISCUSSION THREAD

**Discussion by nicholasvafiades** [xanderfrangos/twinkle-tray#323] [Ideas]
It would be great if there was a way to incorporate sunrise/sunset times in the Time of Day Adjustments (based on location and that automatically vary with changing sunrise/sunset times throughout the year). You could also do Sunset +/- a certain fixed time as well and set fade times between 2 brightness levels over a time interval (changing at fixed 1% increments evenly between those times). We can brainstorm the UI for this if more people support this idea.

**Comment by t583** [comment]
I would love this as well! Currently I have like 5 time Slots in order to make my screen work with my well lit room. These will need to be adjusted constantly throughout the year
